script/pageload/body.js

```diff
@@ -11,25 +11,41 @@ var body = function(){
         setThumbnail('');
     }
 
-    var isOnUserProfile = isOnGitHub && url.match(/.+\/.+\/?/gi) != null;
+    var isOnUserProfile = isOnGitHub && url.match(/.+\/.+\/?/gi) != null && document.getElementById("report-block-modal");
     var isOnProfile = isOnGitHub && url.match(/.+\/.+\/.+\/?/gi) != null;
     var isOnRepo = isOnGitHub && url.match(/.+\/.+\/.+\/.+\/?/gi) != null;
     var isOnIssuePage = isOnGitHub && ( url.indexOf('/pull/') != -1 || url.indexOf('/issue/') != -1 || url.indexOf('/issues/') != -1 );
+    var isOnIssuesPage = isOnGitHub && url.indexOf('/issues') != -1;
```

Comment:
this reminds me, i should refactor `isOnIssuePage` to be more descriptive of where the user actually is

Reply:
definitely, this is something I can add too. A simple "check if element on page" should suffice, unless you have something else in mind